tasks/podman.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Install podman package.
+  package:
+    name: podman
+    state: present
+
+- name: Check homedir of retrace user.
+  shell: grep retrace /etc/passwd | cut -d ':' -f6
+  register: retrace_home
+
+- name: Set home directory for retrace user.
+  command: usermod retrace -d /var/lib/retrace
+  when: retrace_home.stdout != "/var/lib/retrace"
+
+- name: Check if subuid is set for retrace user.
+  shell: grep retrace /etc/subuid | cat
```

Comment:
What’s with the piping to `cat`?

Reply:
If the grep is empty, the task will fail. `cat` makes it not fail.

Reply:
Can we comment/document that, please? I wouldn't have guessed.

Reply:
I've added a comment.

Reply:
Well, now it’s gone altogether. I assume it’s because it’s provided by the `setup` package and should always be present?